docs/source/tutorials/semantic_image_search.ipynb

```diff
@@ -0,0 +1,1222 @@
9T0MN2hODUavjjz4o0y9nLoZ5Heetr7E0a8akc0JZo/ei6kmmBxuNRSgs17mVSpKE3FvVM77J2djKph4VJKUtHxXMDiVZ8UWnWWW0RWw86cnZXjwaMkTQv3rYjNq/UzsTxAs2+LM60n3FRX/AHX+BZ66mdbN3M1wyso8uJdFI8S6CrFkVRKAsjSJSJdFRpE0WpmtC6bCLEkEoolaFiq3LrYIsi6KRLoCVYurFUTbUCyLIhLmWCpObEzyUazTs9kdKRxY+LdGpbhJMzeY1jxXlXCepDCKPbw03UwdGT+lb4HWeb2dO9DLfVTTt0PTsSc0vEVZBZoho0yoyGWaIAoyCzRUCCGSQBBDJIYRVlDTmVYGbKyL8SsgrJlZF3YzkQUZWSLMowrOp5WYG9TyswIAAAuAALQV5pPmdFSMcj0RhT+cj6nRU8jOWd/aOHUv7xz2FiTehhKleLlHKltds6u7nsLHb8m1vpU/i/yHybW+lT+L/IDisLGtehOhJRnbXZozAiwsSAIsLEgCLCxIAiwsSAIsLEgCLCxIAiwsSAIsLEgCLCxIAiwsSAIsLEgCLCxIAiwsSAIsLEgCLCxIAiwsSAIsLEgCLCxIAiwsSAIsLEgCLCxIAiwsSAIsLEgDMAACJeVkkS8rAqAAIn5WZmk/KzMAWWzKkrcK9f8A/PS/XHF8Vc+nPicDiHhcVCqlez25ntvt3T5pa9Sb0PZqeSXoz5TF1Epux1Ve2q0rqOVJ6bHl1JOTu+JnmnA5XO/sZt46KXJnnXN8JifZ6udK+jRqj6erJQpN1JRTel2rCtRjiaVpSajv6ngVsZ31SNSSu0tn5fgax7VxSTSlTVlpdGLFl037MqXx3dWSUb2se1Y+c7Hq/wD9QzTaV07n0Oe+xn0i3dW4ngYOnGePxWaVkru//ke45Hz2Hr06OLxXeNpTTS0vrcsu0enVg6dFqPiaV81znoxnXxFKWa8EuG1zCljYSrpK74JvT4HVhqkIYyaTSjKN0rWsRXoRWW+iXoSUU1JXRyYzFyotQyO89IuLA3xNaFGClK7d9EtxRrqpTzuLhbhLc8erXp4il32dxqQ4X8xGGxEpVbTqOLkt+DfIuke3Kraai1o1vcrCvGorw1PGlipSk6VV5bPRnb2bfuZt2tna05hXZCpKSeaDi07b7lsxRzSaXFhSvoncgpWfio/9z/8A5kXbRlVd6lD+d/8A+ki9yg21qtdlb3i7tqyrepF0rIItci5WUlGN27IrCpCpG8HdIDg7Y/yf/L+x5db5z3L8D0+1mn3Vnz/seZW+c9y/BG4igANIlAXsAJ2JjxISb4E6p6oCz2CeguVuBPElFS8VdgWjG+7NFHgWpwuaZLEXStNZQ55fcdFCipyszqWCpbuNyjmw1Xd6uxtUSlsdNOjCC8MUiJRQR5lWnZnPKB6dSmmtjjqQysisIvmXRWUSIvXXcDpqO1Rlc26snF7p7MVfnGVTJjxFy91ZVcLp3lBvrHiilOtraXxOxPVNaMpVo063DJU5rZmmSnVlBqUJNPoephe0IVPDW8MvpLZngN1MPNwknob05qaugPp4z0s7NcHwZqkrI+ew2Mq0Ha+aH0WerhcVCvHweGXGLYHRKPi0djCpQjK+iT5o3UlJu+j5Mhq4HBWwzhHNHVGLWmx6jXPYp3ce7cUvCwPN4GVX5qf8rOueHlG63XBnPWhahPXaLA8mJdFI8S6CpRZEKxZIC0S8SsS6RUWRotjNGiYRZIskVTLRegRNtUXSKp3NIxZQRdExii6aW3xAKPPQt6KxVXbNIoKhIuolkibARY5q9PPRxEb20v8ADU67HPUaUMQ3tb+xm8xqcV89JWBapuVQHX2f+0x5cT3jy+zoJYZztq6iV+lj1CTml4iGVaLkG2VGUZoyrQGd2hZMllWBVoi5a/MNAUZBZoo7kQsRYsVYENGc1fY01KyWgGLiVlE0uQwrCS1sZtG8kUaRBz1VaDOc66yfdyOQKAAC4AAvT+cj6nRU8jOen85H1Oip5Gcc/dHDqe+MDrf+GL+c5Drf+GL+c7O7kAAHXjfmcN/J/ZHIdeN+Zw38n9kcgEqLeybuMk82XLLNytqephq9TD9iyqUmlLvLJ2vY0xGOrw7Ow9aMkqs7qUsqu7AeM007NWZLhJJNxaT4tHuypQr9o4SdSKblSzPqyJYqlJ1oYjGQqU5JpQ7trKwPJnhKkMLDENrLN2S4mKhJptRbS4pHrYjFVl2RRkp6zbjLwrVGmGlXoLDwrYuFNNK1LJdteoHiJNuyV2dlPCReAr15qanBpJbI9DJHDVsfXpRWeCWXTa5jHE1cT2PiHWlmcZJJ2sB5ShJxzKLa52NsPhJ4iFScWkqcbu/E9bDSr0ZUKdfFwhdK1LJe69SaVSdGpj4U3ljTWaKSWjA8JRlJ2jFt8kiGmnZqzPTwU8T3VXEPEwoQlLxTcU236E9sKMqOGqqSnKSac0rZgPMUZSdlFu/JBQk20ottbqx6s8VVw3ZOGdGSi5XTdrmuOxtei8M6bUXUipT8K1A8vC0qNSU1XqOnaN1ZXuzFRbeibvtZHvZVHtTFZVa9G79TmhiKmG7FpzpNRk5tXtcDylCbllUZX5WItZ24ntYzHV6WFw1SnJRnUjecsqu7G3dU5dqd7JK6oqe3HmB4EoSj5otX5og9qpiaNTDVqeIxkK2ZXglTaszxQN1hKjwbxN1kUstuJhY9eniq0OxM8Z+JTyLRbciY0flLDYaa1nCWSfoB5GV8mJRlHzRa9Ue5TxLlicbOm1alTtD3HPTrTxvZuJ9oak6dnGVkrAeXGEpJuMW7ckQfQVZxwkaFOnioUIqKbi4Xze88ntKVGeLlPDyThJXdlbUDlAAAAAAAAAAAAAAABpQoVcRUyUYOUjTE4HEYWKlVp2i+Kd0dWEbh2RiZ03aeZJtb2GAk5dn4yM23TUbq/BgeaAAN/ZK3svtKjene2j1RlTpzq1FCnFylLRJHRgcbPCVHpmpS88HxO2pi
```

Comment:
This App window didn't render, maybe you didn't call session.freeze() here? 
![image](https://user-images.githubusercontent.com/21222883/213807683-6b6f43ed-095a-49db-bb17-ccd00714c441.png)